### ISSUE TITLE
is there any plan to upgrade react dependency to 17 ?

### ISSUE THREAD

**Issue by passos** [praneshr/react-diff-viewer#118]
the library now depends on react 15 & 16. it doesn't work under react 17. I am wondering is there any plan to upgrade the dependency?

**Comment by rushi** [comment]
Maybe some can help out here if they have ideas?

**Comment by ramkumarkb** [comment]
Hello,

Is this just a matter of adding `^17.0.0` in the `peerDependencies` or something more than that? 

```
	"peerDependencies": {
		"react": "^15.3.0 || ^16.0.0 || ^17.0.0",
		"react-dom": "^15.3.0 || ^16.0.0 || ^17.0.0"
	},
```

**Comment by feddyups** [comment]
yes, https://github.com/praneshr/react-diff-viewer/pull/125/files

**Comment by tenacioustechie** [comment]
Is there any progress on updating to support react 17?
I noticed the last release was a year ago so I'm wondering the project is active?

**Comment by jmikrut** [comment]
Bump - we need this to correct an issue with legacy peer dependencies in [Payload](https://payloadcms.com).

Can we get the above PR merged in? Everything appears to work with React 17.

**Comment by feddyups** [comment]
it doesnt feel like the project is active, I am considering forking this...

**Comment by tenacioustechie** [comment]
I would help test/contribute if you do @feddyups

**Comment by jonwardopenai** [comment]
I would also help test/contribute.

**Comment by oswaldoacauan** [comment]
We are using it with react@18 and seems to work fine. Maybe just remove the peerDependencies restriction?

**Comment by herndlm** [comment]
there seem to be multiple forks already now with https://www.npmjs.com/package/react-diff-viewer-continued being the most popular so far. I think I'm considering switching to that fork for now, maybe the community should consider the same / agree on a fork

**Comment by faisal004** [comment]
Thanks @herndlm
